content/applications/hr/referrals.rst

```diff
@@ -103,7 +105,7 @@ Share job positions
 ===================
 
 In order to earn referral points, a user must first share a job position so the applicant can apply
-for the position. There are several ways to share job positions within the Referrals application,
+for the position. Job positions can be shared in several ways within the *Referrals* application,
 through the :ref:`View Jobs <referrals/view-jobs>` button and the :ref:`Email A Friend
 <referrals/email-jobs>` button.
```

Comment:
where in the application are these buttons found? a little specificity here would be helpful